pysd/py_backend/external.py

```diff
@@ -0,0 +1,962 @@
+"""
+These classes are a collection of the needed tools to read external data.
+The External type objects created by these classes are initialized before
+the Stateful objects by functions.Model.initialize.
+"""
+
+import re
+import os
+import warnings
+import pandas as pd
+import numpy as np
+import xarray as xr
+from . import utils
+
+try:
+    # Optional dependency as openpyxl requires python 3.6 or greater
+    # Used for reading data giving cell range names
+    from openpyxl import load_workbook
+
+except Exception:
```

Comment:
can you  make this exception more specific? Probably not too important in this context as there is only one line of code in the `try` block, but still good practice.

Reply:
I tried to make an import module exception but the exception is different in python2 and python3 and was crossing.
Don't mind too mutch because in 1.0.0 openpyxl will be a requirement and this try/except will disappear.

Reply:
perfect.